### PR TITLE
Mention renaming of CC properties in upgrade guide

### PR DESCRIPTION
**Pull request by abstratt**
Fixes: #24433

### CODE REVIEW

**Comment by abstratt** [comment]
@bot-gradle test BD

File: subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc [hunk 1]
```diff
@@ -73,6 +73,32 @@ beforeSettings { settings ->
...
+|`org.gradle.configuration-cache`
+
+|`org.gradle.unsafe.configuration-cache-problems`
+|`org.gradle.configuration-cache.problems`*
+
+|`org.gradle.unsafe.configuration-cache.max-problems`
+|`org.gradle.configuration-cache.max-problems`
+|===
+
+Note that the original `org.gradle.unsafe.configuration-cache...` properties continue to be honored in this release (if their corresponding finalized `org.gradle.configuration-cache...` counterparts are not provided),
```
Comment (bamboo) [inline subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc:99]: ❓ I don't understand the relevance of the sentence in parenthesis so I'd leave it out.

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -472,6 +472,9 @@ Configuration cache was introduced as an experimental feature back in [Gradle 6.
 
 To learn how to benefit from this feature, refer to the [Configuration Cache](userguide/configuration_cache.html) documentation.
 
+See [the upgrading guide](userguide/upgrading_version_8.html#configuration_caching_options_renamed) if you were already using this feature in previous releases,
```
Comment (bamboo) [inline subprojects/docs/src/docs/release/notes.md:475]: 👍

**Comment by abstratt** [comment]
@bot-gradle test and merge

**Pull request merged**